.github/copilot-instructions.md

```diff
@@ -71,6 +71,10 @@ In general, whenever a code refers to `@azure/core-*` packages, we will expect c
 
 If a change requires updates to the core packages, you will remind the user to run `pnpm build --filter=@azure/<package-name>...` commands.
 
+### Pre-requisites
+
+- To use MCP tool calls, user must have Powershell installed. Provide [Powershell installation instructions](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.5) if not installed, and recommend restarting VSCode to start the MCP server.
```

Comment:
Need to remove `en-us` from the link since otherwise CI complains
```suggestion
- To use MCP tool calls, user must have Powershell installed. Provide [Powershell installation instructions](https://learn.microsoft.com/powershell/scripting/install/installing-powershell?view=powershell-7.5) if not installed, and recommend restarting VSCode to start the MCP server.
```